src/main/kotlin/graphics/scenery/volumes/VolumeManager.kt

```diff
@@ -828,5 +828,29 @@ class VolumeManager(
     companion object {
         /** Static [ForkJoinPool] for fill task submission. */
         protected val forkJoinPool: ForkJoinPool = ForkJoinPool(max(1, Runtime.getRuntime().availableProcessors()))
+
+        fun regenerateVolumeManagerWithExtraVolume(volume: Volume, hub: Hub ) {
```

Comment:
This is probably the spot to do what you did in SceneryBase 👍

Reply:
Yeah that was not correct, the constructor of `Volume` should actually be the correct place as mentioned above 👍